routes/tweetRoutes.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+const router = express.Router();
+const {createTweet , getTweets , deleteTweets , updateTweets , likeTweets , unLikeTweets} = require('../controllers/TweetController');
+const { ensureAuthorizated } = require('../middeware/authorization')
+
+
+
+
+router.get('/' , getTweets);
+
+router.post('/tweets', ensureAuthorizated, createTweet);
+
+router.patch('/update', ensureAuthorizated,  updateTweets);
+
+router.delete('/delete',  ensureAuthorizated, deleteTweets);
+
+router.put('/likes',ensureAuthorizated ,likeTweets);
+
+router.put('/unlikes',ensureAuthorizated ,unLikeTweets);
```

Comment:
http://expressjs.com/en/5x/api.html#router.use

Reply:
Instead of having to add `ensureAuthorized` on every route, you can add it at top-level